pkgs/development/tools/build-managers/bazel/buildifier/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "buildifier";
+  version = "588d90030bc8054b550967aa45a8a8d170deba0b";
```

Comment:
Please you day of commit as version, because this can be numerical sorted (needed for nix-env) and is human-readable. `rev` then will be the sha256